presto-docs/src/main/sphinx/functions/geospatial.rst

```diff
@@ -58,6 +58,17 @@ Constructors
 
     Returns a geometry type polygon object from WKT representation.
 
+.. function:: ST_SphericalGeometryFromGeometry(Geometry) -> SphericalGeometry
```

Comment:
`ST_*` names are reserved for functions from the SQL/MM Part 3 specification. Let's use `to_spherical_geometry` and `from_spherical_geometry` here.